arraycontext/context.py

```diff
@@ -463,6 +464,15 @@ def compile(self, f: Callable[..., Any]) -> Callable[..., Any]:
         """
         return f
 
+    def force_evaluate(self, array: Array) -> Array:
```

Comment:
Thanks for adding a type annotation. Technically, this should now be `ArrayOrContainerT`, and definitely a type variable.  I saw that `freeze` and `thaw` also have incorrect annotations, we should fix those too.

Reply:
Thanks for having a look @inducer!   I think I've addressed your comments in ([2f142f8](https://github.com/inducer/arraycontext/pull/180/commits/2f142f8b989b8f2c9d47c3a08f787abb5c79863b)).   I tried to annotate with `ArrayOrContainerT`, but there was a circular import issue.